CalcualtorLogic.js

```diff
@@ -23,14 +23,19 @@ calculatorLogic.subtract();
 calculatorLogic.multiply();
 calculatorLogic.divide();
 
+
 class Calculator {
+
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
+        this.resultDiv = document.querySelector();
```

Comment:
Incorrect, the `resultDiv` can't be queried like this.
because it is not exist at this point.
you're creating the  resultDiv, so you dont need to query it

![image](https://user-images.githubusercontent.com/16171515/109382425-46bd1d00-78d8-11eb-90c7-ae5a1c2465de.png)


in the `render` method, after creating the `resultDiv` 
save it as property on the class